pkg/vere/pier.c

```diff
@@ -7,9 +7,11 @@
 #include "version.h"
 
 #define PIER_READ_BATCH 1000ULL
-#define PIER_PLAY_BATCH 500ULL
 #define PIER_WORK_BATCH 10ULL
 
+/// The default replay batch size. This can be overridden at the command line.
+static c3_d replay_batch_sz_d = 500ULL;
```

Comment:
Is this 500 events (guessing ULL means unsigned long long?) - thoughts on bumping that up to 100k events? When I've run replays 100k events is usually a fine batch (typically you can do a couple hundred k without issue, but 100k seems good for default?)

Reply:
Yep, `ULL` means `unsigned long long`. Because you can override the batch size from the command line with `--batch-size`, I'm inclined to leave the default at 500 to minimize the risk associated with this change.